core/src/main/java/com/microsoft/applicationinsights/web/internal/correlation/CdsProfileFetcher.java

```diff
@@ -122,7 +122,8 @@ public ProfileFetcherResult fetchApplicationId(String instrumentationKey, Teleme
             if (cause != null && cause instanceof SSLHandshakeException) {
                 throw new FriendlyException("ApplicationInsights Java Agent failed to connect to CdsProfile end point.",
                         "Unable to find valid certification path to requested target.",
-                        "Please import the SSL certificate from "+ Defaults.PROFILER_ENDPOINT +", into the java key store.",
+                        "Please import the SSL certificate from "+ Defaults.PROFILER_ENDPOINT +", into the java key store. "+
+                        "Learn more about importing the certificate here: https://go.microsoft.com/fwlink/?linkid=2151450",
```

Comment:
Can we link directly to the anchor: https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-troubleshoot#ssl-certificate-issues?

Reply:
updated the url to point to ssl certificate issues section